src/Symfony/Component/Messenger/Tests/Transport/Doctrine/DoctrineIntegrationTest.php

```diff
@@ -17,22 +17,27 @@
 use Symfony\Component\Messenger\Transport\Doctrine\Connection;
 
 /**
- * @requires pdo_mysql
+ * @requires extension pdo_sqlite
  */
 class DoctrineIntegrationTest extends TestCase
 {
     private $driverConnection;
     private $connection;
 
-    protected function setUp()
+    /**
+     * @after
+     */
+    public function cleanup()
     {
-        parent::setUp();
+        @unlink(sys_get_temp_dir().'/symfony.messenger.sqlite');
+    }
 
-        if (!getenv('MESSENGER_DOCTRINE_DSN')) {
-            $this->markTestSkipped('The "MESSENGER_DOCTRINE_DSN" environment variable is required.');
-        }
-        $dsn = getenv('MESSENGER_DOCTRINE_DSN');
-        $this->driverConnection = DriverManager::getConnection(['url' => $dsn]);
+    /**
+     * @before
+     */
+    public function createConnection()
+    {
+        $this->driverConnection = DriverManager::getConnection(['pdo' => new \PDO('sqlite:'.sys_get_temp_dir().'/symfony.messenger.sqlite')]);
```

Comment:
@nicolas-grekas We can no longer test locally with other drivers (Mysql / Postgres for example). 

Maybe like this ? 
```suggestion
        if (getenv('MESSENGER_DOCTRINE_DSN')) {
            $this->driverConnection = DriverManager::getConnection(['url' => $dsn]);
        } else {
            $this->driverConnection = DriverManager::getConnection(['pdo' => new \PDO('sqlite:'.sys_get_temp_dir().'/symfony.messenger.sqlite')]);
        }
```

Reply:
agreed, updated